main.tf

```diff
@@ -82,3 +71,19 @@ resource "aws_elasticache_replication_group" "this" {
     Name = local.name
   })
 }
+
+###############################################
+# DNS
+###############################################
+data "aws_route53_zone" "selected" {
+  zone_id = var.route53_zone_id
+}
+
+resource "aws_route53_record" "elasticache" {
+  count   = local.create_route_53_cname_record ? 1 : 0
+  name    = local.name
```

Comment:
this should be a separate var. DNS entries should never be implicit for safety reasons (and because users will typically want to confirm that)

Reply:
okay, it's a var